src/components/forms/BookLoanForm.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from "react";
+
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+import Grid from "@mui/material/Grid";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Select from "@mui/material/Select";
+import MenuItem from "@mui/material/MenuItem";
+import FormControl from "@mui/material/FormControl";
+import InputLabel from "@mui/material/InputLabel";
```

Comment:
```suggestion
import {
    InputLabel, FormControl, MenuItem, Select, Typography, Box, Grid, TextField, Button
} from "@mui/material";
```
we can combine these imports. In this way we can reduce our build size as well. We can discuss its further details over call

Reply:
sorry about this. i was wrong about the build size. the used notation is correct